hieradata/common.yaml

```diff
@@ -683,6 +683,7 @@ govuk::apps::smokey::rate_limit_token: "%{hiera('smokey_rate_limit_token')}"
 
 govuk::apps::static::redis_host: "%{hiera('sidekiq_host')}"
 govuk::apps::static::redis_port: "%{hiera('sidekiq_port')}"
+govuk::apps::static::unicorn_worker_processes: "8"
```

Comment:
This will also need to be added to the `hieradata_aws/common.yaml` file for integration (and for when our apps are all in AWS in the future).

Reply:
Thanks @thomasleese! Have amended.